zip-0226.rst

```diff
@@ -325,6 +325,111 @@ A ZSA Asset Burn description is encoded in a transaction as an of a AssetBurn ty
 | 8     | valueBurn | :math:`\{1 .. 2^{64} - 1\}` | The amount of the AssetType being burnt.                                                                                  |
 +-------+-----------+-----------------------------+---------------------------------------------------------------------------------------------------------------------------+
 
+TxId Digest
+===========
+
+The transaction digest algorithm defined in ZIP 244 [#zip-0244]_ is modified by the ZSA protocol to add a new branch for issuance information, and add an additional hash for the Asset Base within the ``orchard_digest``. The details of these changes are described in this section. The overall tree structure of the hash is as follows, and the additional names not referenced in ZIP 244 are described in detail below::
+
+    txid_digest
+    ├── header_digest
+    ├── transparent_digest
+    │   ├── prevouts_digest
+    │   ├── sequence_digest
+    │   └── outputs_digest
+    ├── sapling_digest
+    │   ├── sapling_spends_digest
+    │   │   ├── sapling_spends_compact_digest
+    │   │   └── sapling_spends_noncompact_digest
+    │   ├── sapling_outputs_digest
+    │   │   ├── sapling_outputs_compact_digest
+    │   │   ├── sapling_outputs_memos_digest
+    │   │   └── sapling_outputs_noncompact_digest
+    │   └── valueBalance
+    └── orchard_digest
+    │   ├── orchard_actions_compact_digest
+    │   ├── orchard_actions_assetid_digest
+    │   ├── orchard_actions_memos_digest
+    │   ├── orchard_actions_noncompact_digest
+    │   ├── flagsOrchard
+    │   ├── valueBalanceOrchard
+    │   └── anchorOrchard
+    └── issuance_digest
```

Comment:
Let's do just top level here: 
```
txid_digest
    ├── header_digest
    ├── transparent_digest
    ├── sapling_digest
    └── orchard_digest
    └── issuance_digest
```

Reply:
Actually let's remove the big tree completely. Seems redundant.

Reply:
I updated the tree, with the `[UPDATED FOR ZSA]` labels. Thought it seemed good for completeness to have the whole table mentioned once. 

Maybe we could remove the issuance version of the tree from the Issuance ZIP (ZIP 227), since that is repeated information from the Transfer ZIP (ZIP 226).

Reply:
Removed.